### PR TITLE
Multiarch-manager-operator: Add jobs for release-4.16 branch

### PR DESCRIPTION
**Pull request by Prashanth684**

### CODE REVIEW

File: ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    namespace: origin
+    tag: release-4.16
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
```
Comment (aleskandro) [inline ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml:53]: are you doing this to enable branching? Don't we need it in the main branch too?

File: ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+promotion:
+  to:
+  - excluded_images:
+    - operator-sdk-src
+    namespace: origin
+    tag: release-4.16
+releases:
+  initial:
+    integration:
+      name: "4.15"
```
Comment (aleskandro) [inline ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml:49]:
```suggestion
      name: "4.16"
```

also, would we start pivoting to 4.16? We'd need this done for the main branch too.

Comment (Prashanth684) [reply ci-operator/config/openshift/multiarch-manager-operator/openshift-multiarch-manager-operator-release-4.16.yaml:49]: right...4.16 is better unless we plan on creating a release-4.15

**Comment by Prashanth684** [comment]
/pj-rehearse pull-ci-openshift-multiarch-manager-operator-release-4.16-images

**Pull request merged**